spec/controllers/users_controller_spec.rb

```diff
@@ -0,0 +1,37 @@
+require 'rails_helper'
+
+RSpec.describe '/users', type: :request do
+  describe 'UsersController' do
+    context 'GET index' do
+      before(:example) do
+        get '/users'
+      end
+
+      it 'success for index action' do
+        expect(response).to have_http_status(:success)
+      end
+
+      it 'Render correct templete for index action' do
+        expect(response).to render_template(:index)
+      end
```

Comment:
- [ ] As required for the project, kindly ensure that your index specs include tests that ensures the response body includes the correct placeholder text. Example : `expect(response.body).to include("Welcome to user index page")`